target/linux/kirkwood/patches-5.4/997-device_tree_cmdline.patch

```diff
@@ -0,0 +1,12 @@
+--- a/drivers/of/fdt.c
++++ b/drivers/of/fdt.c
```

Comment:
A (short) comment or header on that patch could be helpful.

Reply:
You're right, I'll add it.

Reply:
Maybe we'd better move the patch to generic (as now three targets use it), but I'm not sure if it's safe...